Dockerfile

```diff
@@ -0,0 +1,32 @@
+##### Run kartograafr in Docker.
+# TODO: - a couple of settings should be overrideable by environment variables.
+#       - This runs as root.  Cron may require further work if the root user is not allowed.
+
+FROM python:2.7
+
+RUN apt-get update && apt-get install -y cron
+
+######### set timezone
+# TODO: allow overriding the time zone setting with an environment variable.
+ENV TZ=America/New_York
```

Comment:
My thinking has evolved on this.  IMHO Stored time should always be stored with a timezone.  Internal printed time (e.g. in a log) should always print the timezone.  This means that time is essentially in UTC but also easy to read and unambiguous. (Time displayed to a user requires more case by case analysis since you don't want to display timezone clutter but can't assume the user is in the server's timezone.)

Reply:
FWIW, UTC expressed as ISO 8601 must have a timezone of either "Z", "+00:00", "+0000", or "+00".